src/main/kotlin/bancoDeDados/ERD/queries/queries.sql

```diff
@@ -0,0 +1,203 @@
+GROUP BY product_tech.tribo_name;
+
+-- ex 11
+-- Para cada Tribe, quantas séries cada squad assistiu (Cuidado para não contar a mesma série duas vezes numa mesma squad)
+SELECT
+    product_tech.tribo_name,
+    squad.squad_name,
+    COUNT(DISTINCT series.serie_name) AS qtd_series_assistidas
+FROM tripulante
+INNER JOIN series_assistidas ON tripulante.matricula = series_assistidas.matricula
+INNER JOIN series ON series.serie_id = series_assistidas.serie_id
+INNER JOIN tripulante_squad ON tripulante.matricula = tripulante_squad.matricula
+INNER JOIN squad ON squad.squad_id = tripulante_squad.squad_id
+INNER JOIN product_tech ON squad.tribo_id = product_tech.tribo_id
+GROUP BY product_tech.tribo_name, squad.squad_name;
+
+-- ex 12
+-- Para cada Tribe, com exceção da Squad de top-up, quantas séries cada squad assistiu
+USE TRIBE_EXERCISE;
+SELECT
+    product_tech.tribo_name,
+    squad.squad_name,
+    COUNT(DISTINCT series.serie_name) AS qtd_series_assistidas
+FROM tripulante
+INNER JOIN series_assistidas ON tripulante.matricula = series_assistidas.matricula
+INNER JOIN series ON series.serie_id = series_assistidas.serie_id
+INNER JOIN tripulante_squad ON tripulante.matricula = tripulante_squad.matricula
+INNER JOIN squad ON squad.squad_id = tripulante_squad.squad_id
+INNER JOIN product_tech ON squad.tribo_id = product_tech.tribo_id
+where not squad.squad_name = 'topUp'
+GROUP BY product_tech.tribo_name, squad.squad_name;
+
+-- ex 13
+-- Quem são as pessoas Cross da Tribe (Pessoas que atuam em mais de uma Squad)
+USE TRIBE_EXERCISE;
+SELECT nome
+FROM tripulante
+INNER JOIN tripulante_squad ON tripulante_squad.matricula = tripulante.matricula
+GROUP BY tripulante.matricula
+HAVING COUNT(DISTINCT tripulante_squad.squad_id) > 1;
+
+-- ex 14
+-- Quantas vezes cada série foi assistida por pessoas da Creditas
+USE TRIBE_EXERCISE;
+SELECT serie_name, COUNT(series_assistidas.serie_id) AS vezes_assistida
+FROM series
+LEFT JOIN series_assistidas ON series.serie_id = series_assistidas.serie_id
+GROUP BY series.serie_id;
+
+-- ex 15
+-- Quais as 3 séries mais assistidas por pessoas da Creditas e quantas vezes cada uma foi vistas
+USE TRIBE_EXERCISE;
+SELECT serie_name, COUNT(series_assistidas.serie_id) AS vezes_assistida
+FROM series
+LEFT JOIN series_assistidas ON series.serie_id = series_assistidas.serie_id
+GROUP BY series.serie_id
+ORDER BY vezes_assistida DESC LIMIT 3;
+
+-- ex 16
+-- Para cada Squad, qual o cargo que assistiu a maior quantidade de séries diferentes e quantas foram
+-- (Ex: Se todas as pessoas desenvolvedoras assistiram a mesma série, então o cargo desenvolvedor só assistiu
+-- 1 série diferente)
+SELECT
+  squad_name,
+  cargo,
+  series_assistidas
+FROM (
+  SELECT
+    squad.squad_name,
+    tripulante.cargo,
+    COUNT(DISTINCT series_assistidas.serie_id) as series_assistidas,
+    RANK() OVER (PARTITION BY squad.squad_id ORDER BY COUNT(DISTINCT series_assistidas.serie_id) DESC) AS verificar
```

Comment:
@Julia-Mayumi , @rkataokaAtCreditas Uma função que encontrei na internet foi sobre o RANK, em tese ele percorre em cada squad e o cargo e vai verificando as series assistidas, o partition faz a divisão e individualização de cada resultado por squad, nesse link [aqui](https://www.sqlservertutorial.net/sql-server-window-functions/sql-server-rank-function/) tem uma explicação melhor de como funciona, mas caso tenha uma sugestão melhor para a query estou a disposição :green_heart:

Reply:
Caramba André o_o
Parabéns pela pesquisa :confetti_ball: 

Eu não conhecia essa função do banco :thinking: 
Ela funciona muito bem para esse exercício, mas você deve ter percebido que ela tem uma desvantagem :eyes: 
Que ela te força a fazer dois `counts` na mesma query né?
Mas como o intuito é praticar queries de métricas, sua resposta está certa :confetti_ball: 
Com a ressalva de se fossemos usar essa query em algum sistema, teriamos que melhorar ela para ser mais performatica, blz? Pois fazer dois `count` pode ser bem oneroso para o banco se ele tiver bastante dados

Reply:
Ai aqui está a resposta que eu estava esperando nesse exercício:
``` sql
with tab as (
    SELECT squad.squad_name as squad_name,
    tripulante.cargo as cargo,
    COUNT(DISTINCT series_assistidas.serie_id) as quant_series_assistidas
	    FROM squad
    JOIN tripulante_squad ON squad.squad_id = tripulante_squad.squad_id
    JOIN tripulante ON tripulante_squad.matricula = tripulante.matricula
    LEFT JOIN series_assistidas ON tripulante.matricula = series_assistidas.matricula
    GROUP BY squad.squad_id, tripulante.cargo
) 


select DISTINCT ON(tab.squad_name) tab.squad_name,
    tab.cargo,
    tab.quant_series_assistidas as maior_número_de_series_assistidas
from tab
order by tab.squad_name, tab.quant_series_assistidas desc
```

Basicamente o que eu fiz de diferente de você foi usar o `distinct on`, e ordenar depois pra ele só trazer 1 cargo para squad que assistiu mais series :upside_down_face:

Reply:
Excelente! Eu também não tinha pensado no uso RANK pra esse caso, mas o raciciocínio está correto 🔝 

A solução que me veio em mente era usar `MAX(series_assistidas)` e reagregar por `squad_name` e `cargo`.